.github/ISSUE_TEMPLATE/bounty.yml

```diff
@@ -0,0 +1,52 @@
+name: Bounty
+description: Create a bounty
+labels: ["bounty"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bounty
+  - type: textarea
+    id: overview
+    attributes:
+      label: Overview
+      description: General background needed for bounty hunter to complete task
+      placeholder: General background needed for bounty hunter to complete task
+    validations:
+      required: true
+  - type: textarea
+    id: reference
+    attributes:
+      label: Reference
+      description: An explanation as to where the bounty hunter should look or code that is affected
+      placeholder: An explanation as to where the bounty hunter should look or code that is affected
+    validations:
+      required: true
+  - type: textarea
+    id: acceptance-criteria
+    attributes:
+      label: Acceptance Criteria
+      description: The criteria that needs to be completed in order for the bounty to be considered complete
+      placeholder: The criteria that needs to be completed in order for the bounty to be considered complete
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots-or-mockups
```

Comment:
We can exclude this portion from the unchained bounty template. Not really applicable.